src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -292,6 +297,37 @@ public WorkspaceDescription updateWorkspace(
         "Error updating workspace");
   }
 
+  public CloneWorkspaceResult cloneWorkspace(
```

Comment:
nit: public method should have a description. I think including the WSM endpoint path is helpful

Reply:
👍🏻